crates/zombie_actor/src/ghost_channel.rs

```diff
@@ -112,6 +112,12 @@ impl<
                 payload,
                 span: self.span,
             })?;
+        } else {
+            // TODO XXX - if the user is sending back an error
+            // it could get lost here... we are going to panic
+            if let Err(e) = payload {
+                panic!("DISALLOWING ignored errors due to lazy publish: {:?}", e);
```

Comment:
What are you implying when you call the publish _lazy_?

Reply:
haha, updated better text : )